### DISCUSSION TITLE
Welcome to AtlantaBot Discussions!

### DISCUSSION THREAD

**Discussion by Androz2091** [Androz2091/AtlantaBot#354] [General]
## 👋 Welcome!

  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about (hosting, understanding of the code...)
  * Share ideas, new cool features that you would like to see on Atlanta.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction from yourself and tell us what you use Atlanta for!

**Comment by ghost** [comment]
I need some help when I try to go to https://explain.com:8080 (not real URL this is just so I don't reveal my URL) then it redirects me to the discord authorize page but it says "invalid OAuth2 redirect_uri"

**Comment by Harshrwt** [comment]
my bot is not responding

**Comment by Androz2091** [comment]
Please open new discussions for new problems
